### ISSUE TITLE
Z-Image-Base Pre-training alignment reference

### ISSUE THREAD

**Issue by Peterande** [Tongyi-MAI/Z-Image#136]
Very glad to finally get the z-image-base weights! During the pre-training stage, besides the FM loss, are there any other representation-supervision losses? Do you have any recommendations for best-practice fine-tuning hyperparameters/implementation details, such as learning rate, Adam betas, etc.? I found that training only converges a bit faster when I use a learning rate on the order of 1e-4.

**Comment by n0kovo** [comment]
Could you reopen or provide whatever you might have found out?
